qcengine/testing.py

```diff
@@ -116,6 +116,7 @@ def get_job(self):
     "dftd3": which('dftd3', return_bool=True),
     "geometric": which_import("geometric", return_bool=True),
     "psi4": is_program_new_enough("psi4", "1.2"),
+    "psi4_14": is_program_new_enough("psi4", "1.4a2.dev160"),
```

Comment:
I'd call this `"psi4_runschema"` or something instead of `"psi4_14"`. but not terribly important.